code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -65,8 +65,6 @@
 	var/lights_power = 6
 	///Just stop the mech from doing anything
 	var/completely_disabled = FALSE
-	///Whether this mech is allowed to move diagonally
-	var/allow_diagonal_movement = FALSE
```

Comment:
`
this isnt a balance change you idiot, the refactor added this in presumably by accident as it breaks mech strafing (and also doesnt make any sense at all)
wow you didnt even mention the balance change in your own PR, guess this makes even more sense then.
`

If you read the comment that LITERALLY stands above it, which there is no way you couldnt have, it says this is a limiter on what can move diagonally (preserving behaviour for shit like odysseuses that were previously snowflaked)

If this actually breaks strafing then fix it instead of buffing combat mechs

Do not explain to me code that i wrote myself I know it better than you do

Reply:
Following testing I am also unable to reproduce this mysterious bug of yours so I'm asssuming you made that up